cfme/tests/generic_objects/test_instances.py

```diff
@@ -242,8 +243,9 @@ def test_generic_objects_tag_rest(appliance, generic_object, tags):
         test_flag: rest
 
     Polarion:
-        assignee: anikifor
+        assignee: jdupuy
         initialEstimate: 1/4h
+        casecomponent: GenericObjects
     """
```

Comment:
@john-dupuy 
Polarion tag  ```caseimportance:``` is missing.
Please add this in above test also.

Reply:
@ganeshhubale this is taken care of by #8424, so I will just wait until that gets merged.

Reply:
@john-dupuy @lina-nikiforova I found contradiction between `assignee` @valaparthvi or @john-dupuy. please clear this?

Reply:
@digitronik @valaparthvi is the correct assignee, I made a mistake here but am waiting for #8424 to be merged so that I can rebase and push my fixes.